app/views/device_stories/show.html.erb

```diff
@@ -9,11 +8,20 @@
     <%- end %>
   </h2>
 </div>
+<div class="row">
+  <div class="col-sm-12 text-right">
+    <div class="btn-group" role="group" aria-label="Time Range">
+      <button class="btn btn-sm <%= params[:time_range].blank? || params[:time_range] == "1" ? "btn-primary" : "btn-default" -%>" data-behavior="change-time-range" data-time-range="1">2 months</button>
+      <button class="btn btn-sm <%= params[:time_range] == "2" ? "btn-primary" : "btn-default" -%>" data-behavior="change-time-range" data-time-range="2">6 months</button>
+      <button class="btn btn-sm <%= params[:time_range] == "3" ? "btn-primary" : "btn-default" -%>" data-behavior="change-time-range" data-time-range="3">1 year</button>
```

Comment:
Still not a big fan of the 1/2/3 stuff going on here, but at least now it's all in the ruby view layer (before it was also in the JS layer).